router/router.go

```diff
@@ -321,7 +322,7 @@ func (r *Router) serveHTTPS(server *http.Server, errChan chan error) error {
 
 	r.tlsListener = tls.NewListener(listener, tlsConfig)
 
-	r.logger.Info("tls-listener-started", zap.Object("address", r.tlsListener.Addr()))
+	r.logger.Info("tls-listener-started", slog.Any("address", r.tlsListener.Addr()))
```

Comment:
I was doing some manual testing of this PR and I found that this log line changed structure.

BEFORE
```
{
  "log_level": 1,
  "timestamp": "2024-09-09T17:25:48.706106838Z",
  "message": "tls-listener-started",
  "source": "vcap.gorouter.router",
  "data": {
    "address": {
      "IP": "0.0.0.0",
      "Port": 443,
      "Zone": ""
    }
  }
}
```

AFTER
```
{
  "log_level": 1,
  "timestamp": "2024-09-09T20:40:24.065520680Z",
  "message": "tls-listener-started",
  "source": "vcap.gorouter.router",
  "data": {
    "address": "0.0.0.0:443"
  }
}
```

There's a chance something else changed this (other dependency bumps etc). I will look into it more tomorrow.

❓ Do you see this same before/after change on your systems?